README.md

```diff
@@ -473,3 +473,85 @@ Available modes:
 - 0: Print all tracing information. Comprehensive output, including: evaluation time and operator profiling.
 - 1: Print evaluation time. Time taken for each evaluation.
 - 2: Profile individual operator. Identify performance bottleneck within the model.
+
+### 7. GGUF
+
+Currently, we support two new features for the GGUF.
+
+1. Support loading & inference official GGUF models from HugginfaceFace.
+2. Support loading & inference official GGUF models from llama.cpp.
+
+Verified GGUF models from HF:
+
+- [TheBloke/Llama-2-7B-Chat-GGUF](https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF) : q4_0 / q5_0 / q8_0
+- [TheBloke/Mistral-7B-v0.1-GGUF](https://huggingface.co/TheBloke/Mistral-7B-v0.1-GGUF) : q4_0 / q5_0 / q8_0
+- [TheBloke/CodeLlama-7B-GGUF](https://huggingface.co/TheBloke/CodeLlama-7B-GGUF) : q4_0 / q5_0 / q8_0
+- [TheBloke/CodeLlama-13B-GGUF](https://huggingface.co/TheBloke/CodeLlama-13B-GGUF): q4_0 / q5_0 / q8_0
+
+
+Verified GGUF models from Llama.cpp:
```

Comment:
no put GGUF from hf or from llama.cpp in here.

Reply:
fixed